nexus-sdk/src/main/java/io/nexusrpc/handler/OperationHandlerException.java

```diff
@@ -1,42 +1,25 @@
 package io.nexusrpc.handler;
 
-import io.nexusrpc.FailureInfo;
 import org.jspecify.annotations.Nullable;
 
 /** Thrown from a handler for any unexpected error. */
 public class OperationHandlerException extends RuntimeException {
```

Comment:
I'd also consider renaming this to `HandlerException` for consistency with the Go SDK.